### DISCUSSION TITLE
Cupón de atraso

### DISCUSSION THREAD

**Discussion by mpalaciosc** [IIC2133-PUC/2024-1#112] [Tarea 1] [answered]
Holaa, hasta cuando va a estar disponible el form de cupones de atraso?

**Comment by benjavicente** [comment, marked as answer]
Hasta el plazo máximo que se puede optar.

**Reply by tariztia** [reply]
Hola, no sé si entendí bien. O sea como se entrega hoy, va a estar disponible hasta el viernes?

**Reply by sebacarrascop** [reply]
si!, puedes solicitar los tickets hasta el viernes (si usas 2) o el jueves (si solo usas 1)

**Comment by rominaarancibia** [comment]
Solo por confirmar, si uso un cupón no se me aplicaría ningún descuento por entregar mañana hasta las 23:59?

**Reply by benjavicente** [reply]
Sip. El descuesto es luego de los días adicionales dado los cupones.

**Comment by llambias** [comment]
Duda, para estar 100% seguro el form del cupon se puede completar el dia después de la entrega, es decir si la entrega es oficialmente el viernes, puedo llenarlo el sabado? y si lo llenara el sabado, podria el domingo decidir usar el segundo cupon y llenarlo denuevo?

**Reply by benjavicente** [reply]
Sip. El formulario de cupones queda abierto por varios días más.

**Comment by cseverino99** [comment]
Hola! que pasa si respondí que quería tener dos días más con cupones de atrasdos pero ahora la tarea cambió de fecha? puedo enviar denuevo el form?

**Reply by benjavicente** [reply]
Sí. Como dice el form, se toma la última respuesta. Además, está configurado para enviar la respuesta por mail para poder ser cambiada más adelante.

**Comment by czamoraz** [comment]
Hola! ¿Los cupones son para días hábiles y no hábiles?

**Reply by sebacarrascop** [reply]
Si!
